source/mir/random/flex/internal/area.d

```diff
@@ -1022,3 +1052,34 @@ unittest
         }
     }
 }
+
+/**
+Calculate the parameters for an interval.
+Given an interval, determine its type (e.g. purely concave, or purely convex)
+and its hat and squeeze function.
+Given these functions, compute the area and overwrite the references data type.
+
+Params:
+    iv = Interval which should be calculated
+*/
+void calcInterval(S)(ref Interval!S iv)
+in
```

Comment:
`calcInterval` only calls stuff from `area.d`, so I thought it makes sense to move it over.